packages/server/__tests__/api/users.test.js

```diff
@@ -99,13 +99,13 @@ describe('`/api/users` endpoint', async () => {
                 const response = await fetchApi(`/users`, agencies.own, fetchOptions.admin);
                 expect(response.statusText).to.equal('OK');
                 const json = await response.json();
-                expect(json.length).to.equal(11);
+                expect(json).to.have.lengthOf(7);
```

Comment:
I changed the `expect()` style here to give a more verbose and helpful message while I was at it. There’s a *lot* of room to improve the expectations with things like this all over these test suites. The `chai-as-promised` plugin can also help clean things up and provide better messaging around async code as well.